client/components/date-range/index.js

```diff
@@ -430,6 +433,11 @@ export class DateRange extends Component {
 	 * @returns {import('react').Element} the Popover component
 	 */
 	renderPopover() {
+		const { displayShortcuts } = this.props;
+
+		// Only render shortcuts if displayShortcuts is true
+		const shortcuts = displayShortcuts ? this.displayShortcutsComponent() : null;
```

Comment:
Is this necessary? You have the `displayShortcuts` prop so you can conditionalise the rendering below based on that.

Reply:
Addressed in 519fd56ebf